test/e2e/framework/pod/create.go

```diff
@@ -167,11 +166,12 @@ func MakeSecPod(podConfig *Config) (*v1.Pod, error) {
 	if podConfig.NS == "" {
 		return nil, fmt.Errorf("Cannot create pod with empty namespace")
 	}
-	if len(podConfig.Command) == 0 {
+	if len(podConfig.Command) == 0 && !NodeOSDistroIs("windows") {
 		podConfig.Command = "trap exit TERM; while true; do sleep 1; done"
 	}
+
 	podName := "pod-" + string(uuid.NewUUID())
-	if podConfig.FsGroup == nil && runtime.GOOS != "windows" {
+	if podConfig.FsGroup == nil && !NodeOSDistroIs("windows") {
```

Comment:
`runtime.GOOS != "windows"` will always be true because we never run the e2e binary in a windows node

Reply:
y, so here it fixed it, right?

Reply:
if it's always true then:

```
if podConfig.FsGroup == nil && runtime.GOOS != "windows" {
```

would be equal to

```
if podConfig.FsGroup == nil {
```

Reply:
> if it's always true then:
> 
> ```
> if podConfig.FsGroup == nil && runtime.GOOS != "windows" {
> ```
> 
> would be equal to
> 
> ```
> if podConfig.FsGroup == nil {
> ```

But here we use NodeOSDistroIs instead runtime.GOOS?

Reply:
I understand now, thanks!